code/modules/mob/living/carbon/xenomorph/castes/puppeteer/abilities_puppeteer.dm

```diff
@@ -0,0 +1,466 @@
+	owner_xeno.face_atom(target_human)
+	owner_xeno.do_attack_animation(target_human, ATTACK_EFFECT_REDSLASH)
+	owner_xeno.visible_message(target_human, span_danger("[owner_xeno] flays and rips skin and flesh from [target_human]!"))
+	playsound(target_human, "alien_claw_flesh", 25, TRUE)
+	target_human.emote("scream")
+	owner_xeno.emote("roar")
+	target_human.apply_damage(45, def_zone = BODY_ZONE_CHEST, blocked = MELEE, sharp = TRUE, edge = FALSE, updating_health = TRUE, penetration = 20)
+	target_human.Paralyze(0.8 SECONDS)
+
+	owner_xeno.gain_plasma(owner_xeno.xeno_caste.flay_plasma_gain)
+
+	add_cooldown()
+
+// ***************************************
+// *********** Pincushion
+// ***************************************
+/datum/action/xeno_action/activable/pincushion
+	name = "Pincushion"
+	action_icon_state = "pincushion"
+	desc = "Launch a spine from your tail. This attack will help deter any organic as well as support your puppets and teammates in direct combat."
+	cooldown_timer = 5 SECONDS
+	keybinding_signals = list(
+		KEYBINDING_NORMAL = COMSIG_XENOABILITY_PINCUSHION,
+	)
+
+/datum/action/xeno_action/activable/pincushion/can_use_ability(atom/victim, silent = FALSE, override_flags)
+	. = ..()
+	if(!.)
+		return
+	var/mob/living/carbon/xenomorph/X = owner
+	if(X.do_actions)
+		return FALSE
+	X.face_atom(victim)
+	if(!do_after(X, 0.3 SECONDS, FALSE, victim, BUSY_ICON_DANGER, extra_checks = CALLBACK(X, TYPE_PROC_REF(/mob, break_do_after_checks), list("health" = X.health)), ignore_turf_checks = TRUE))
+		return FALSE
+	succeed_activate()
+
+/datum/action/xeno_action/activable/pincushion/use_ability(atom/victim)
+	var/mob/living/carbon/xenomorph/xeno = owner
+	var/turf/current_turf = get_turf(owner)
+	playsound(xeno.loc, 'sound/bullets/spear_armor1.ogg', 25, 1)
+	xeno.visible_message(span_warning("[xeno] shoots a spike!"), span_xenonotice("We discharge a spinal spike from our body."))
+
+	var/obj/projectile/spine = new /obj/projectile(current_turf)
+	spine.generate_bullet(/datum/ammo/xeno/spine)
+	spine.def_zone = xeno.get_limbzone_target()
+	spine.fire_at(victim, xeno, null, range = 6, speed = 1)
+
+	add_cooldown()
+// ***************************************
+// *********** Dreadful Presence
+// ***************************************
+/datum/action/xeno_action/dreadful_presence
+	name = "Dreadful Presence"
+	action_icon_state = "dreadful_presence"
+	desc = "Emit a menacing presence, striking fear into the organics and slowing them for a short duration."
+	plasma_cost = 50
+	cooldown_timer = 20 SECONDS
+	keybinding_signals = list(
+		KEYBINDING_NORMAL = COMSIG_XENOABILITY_DREADFULPRESENCE,
+	)
+
+/datum/action/xeno_action/dreadful_presence/action_activate()
+	var/obj/effect/overlay/dread/effect = new
+	owner.vis_contents += effect
+	for(var/mob/living/carbon/human/human in view(PETRIFY_RANGE, owner.loc))
+		to_chat(human, span_userdanger("An overwhelming sense of dread washes over you..."))
```

Comment:
Should add that it slows marines down when they feel dread. I didn't even know what this does until I code dive and check concept.